test/simplifyExpression/basicsSearch/testSimplify.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const assert = require('assert');
+const math = require('mathjs');
+
+const flatten = require('../../../lib/util/flattenOperands');
+const print = require('../../../lib/util/print');
+
+
+function testSimplify(exprStr, outputStr, simplifyOperation) {
```

Comment:
I love this function!! It's my dream to eventually have something like this for all the tests (#42)